deltachat-ios/AppDelegate.swift

```diff
@@ -143,6 +144,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UNUserNotificationCenterD
         if let notificationOption = launchOptions?[.remoteNotification] {
             logger.info("Notifications: remoteNotification: \(String(describing: notificationOption))")
             increaseDebugCounter("notify-remote-launch")
+            performFetch(completionHandler: { (_) -> Void in })
```

Comment:
can't you just pass nil here?

Reply:
ok, I see now currently you can't but still normally they _should_ be optional. I think it makes sense to fix it, it's our code as far as i can see.

Reply:
hm, we are using the same syntax as for the completion handler we are forwarding from `performFetchWithCompletionHandler`/`didReceiveRemoteNotification` - changing that to `@escaping ((UIBackgroundFetchResult) -> Void)?` results in an error, maybe because of the @escaping (whatever that is for) - so, maybe things are just fine and the empty completion handler is the minor issue :)

Reply:
@escaping seems to be needed https://www.donnywals.com/what-is-escaping-in-swift/

(sorry for pinging you, @escaping :)